templates/rhpam73-kieserver-postgresql.yaml

```diff
@@ -31,10 +31,19 @@ parameters:
   name: APPLICATION_NAME
   value: myapp
   required: true
+- displayName: Maven mirror URL
+  description: Maven mirror to use by Business Central and KIE server.
+  name: MAVEN_MIRROR_URL
+  required: false
+- displayName: Maven mirror of
+  description: Maven mirror configuration for KIE server.
+  name: MAVEN_MIRROR_OF
+  value: "external:*"
```

Comment:
We wanted to default this to "external:*,!repo-custom" because a Maven repo is expected to be used? But if there is a serious use case for NOT configuring a maven repo, let's keep your option